script/clone_all_rspec_repos

```diff
@@ -12,7 +12,11 @@ if is_mri; then
   clone_repo "rspec-core"
   clone_repo "rspec-expectations"
   clone_repo "rspec-mocks"
-  clone_repo "rspec-rails"
+  if requires_rspec_rails_maintenance_branch; then
+    clone_repo "rspec-rails" "4-0-maintenance"
+  else
+    clone_repo "rspec-rails"
+  fi
```

Comment:
Maybe clone both and run against both?

Reply:
The whole point of this is that the build fails against rspec-rails 3-9-maintenance due to the config predicate bug fix. 3.9.x is no longer maintained.